ui/css/failure-summary.css

```diff
@@ -8,6 +8,10 @@
   background: #ffffff;
 }
 
+.failure-summary-push-health-button {
+ margin-bottom: 4px;
+}
```

Comment:
Could you do this using a bootstrap utility instead? https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
sure!